SplunkRumWorkspace/SplunkRum/SplunkRum/NetworkType.swift

```diff
@@ -43,6 +43,13 @@ private func setConnectionType(_ type: String?) {
         pthread_rwlock_unlock(&netInfoLock)
     }
     currentNetInfo.hostConnectionType = type
+
+    if type != "wifi" {
```

Comment:
Should this if type == "wifi"

Reply:
Changed it to `type != `cell` since it might be possible it is nil (no idea in which case though)